app/templates/agenda-items.hbs

```diff
@@ -45,9 +45,15 @@
                 </AuAlert>
               </div>
             {{else}}
+              {{#if @model.count}}
+                <p class="au-u-margin-left au-u-margin-top-large au-u-margin-bottom-small au-u-h4 au-u-medium">{{@model.count}} Zoekresultaten</p>
```

Comment:
I would remove the bottom margin since the text and the items are a bit closer together in the design.

```suggestion
                <p class="au-u-margin-left au-u-margin-top-large au-u-h4 au-u-medium">{{@model.count}} Zoekresultaten</p>
```

Reply:
A border is also shown which isn't there in the design. It comes from here: https://github.com/lblod/frontend-burgernabije-besluitendatabank/blob/93e5cd623536bc42e8cb436464e567f8f8a9f6ac/app/styles/_shame.scss#L207-L209

I would remove it, I doesn't seem to be used anywhere else at the moment (@Dietr?)

Reply:
Both implemented! I moved the border-top definition to its own class for now, in case Dieter still planned on using it

Reply:
Yes, border was there to adapt to the scrolling which is not shown in the design. Without the border the text gets cut of with a white background which looks a bit messy.

Reply:
No worries, will add it back when I do some finetuning of the design. I don't have figma editing permissions but will ask them as it would be easier to align on the design.